### ISSUE TITLE
Add icons to mapping interactions

### ISSUE THREAD

**Issue by jhenshall** [mapswipe/mapswipe#252]
Add icons to the mapping screen interactions to make it clearer what selection has been made. This would improve accessibility and help when colour filters are on (e.g. night mode).

User suggestion is to have an icon preferably on the tile in question showing the status of the tile in a darker form of the colour. So:

- Green tick
- yellow question mark
- red cross

Also, an icon at the next to the word of what you're looking for would be even better. So a building icon, and a road icon would work.

**Comment by laurentS** [comment]
We've had 2 emails from colourblind users who've requested this sort of changes. I've invited them to comment on this ticket, but here are some excerpts from their messages:

> I'm slightly colorblind like many other males and the very minor difference in colors can be confusing. It's nice that i can count my clicks (1=building, 2 = maybe, 3= cloud) but the app would be much easier to use with some sort of other indicator.

and 

> as a colorblind user the shades that occur after the taps are basically indistinguishable for me. Would you please consider adding textures (like cross hatches), icons, and/or alternate color schemes? As the app currently is, I'm unable to use it. About 8% of males and 0.4% of females are colorblind, so improving accessibility would allow more people to use it.

As the ignorant developer, questions from me:
- would icons make more sense than hatches? From the previous comments, something like building/road/whatever we're looking for (see #374 ) | question mark | cloud?
- how big should they be? full tile, small icon in the corner?
- what colour should the icons be? I'm guessing something with high contrast?

Examples from around the web, or screenshots from apps would super helpful!

**Comment by luisgregson** [comment]
An icon sounds like a good idea.

> would icons make more sense than hatches? From the previous comments, something like building/road/whatever we're looking for (see #374 ) | question mark | cloud?
how big should they be? full tile, small icon in the corner?
what colour should the icons be? I'm guessing something with high contrast?

I think it would be good to use numbers in a circle as the icon in the corner because the tutorial already has small number icons to indicate the number of times tapped, so it would be familiar to the user:
![image](https://user-images.githubusercontent.com/8599818/96725460-caa24e00-13a8-11eb-9182-580c3109dda6.png)

However, i've run a contrast test on that icon and it's only 2.45:1 which doesn't meet the WCAG AAA guidelines of 7:1
Maybe some icons like this might work:
![image](https://user-images.githubusercontent.com/8599818/96727611-279f0380-13ab-11eb-94b7-e484ac342c08.png)
![image](https://user-images.githubusercontent.com/8599818/96727748-51582a80-13ab-11eb-900a-33003eee596b.png)
![image](https://user-images.githubusercontent.com/8599818/96727834-69c84500-13ab-11eb-8be5-cea3c0ac8068.png)

The contrast is at minimum 7:1 and the font size is 14 bold, so it meets the WGAC recommendations.

Happy for some feedback on these and i can provide the icons if you want?

**Comment by adrian7031** [comment]
I think an icon (tick, question mark, cross) would be better than digits, but I like the high contrast circles. 
Just to be clear, the square would still be coloured the same as it is now? 
-------- Ursprüngliche Nachricht --------Von: Luis Gregson <notifications@github.com> Datum: 21.10.20  15:43  (GMT+01:00) An: mapswipe/mapswipe <mapswipe@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Betreff: Re: [mapswipe/mapswipe] Add icons to mapping interactions (#252) 
An icon sounds like a good idea.

would icons make more sense than hatches? From the previous comments, something like building/road/whatever we're looking for (see #374 ) | question mark | cloud?
how big should they be? full tile, small icon in the corner?
what colour should the icons be? I'm guessing something with high contrast?

I think it would be good to use numbers in a circle as the icon in the corner because the tutorial already has small number icons to indicate the number of times tapped, so it would be familiar to the user:

However, i've run a contrast test on that icon and it's only 2.45:1 which doesn't meet the WCAG AAA guidelines of 7:1
Maybe some icons like this might work:



The contrast is at minimum 7:1 and the font size is 14 bold, so it meets the WGAC recommendations.
Happy for some feedback on these and i can provide the icons if you want?

**Comment by rachblevine** [comment]
Update: Agree that symbols in addition to colors could really help us here. Other things to consider: checking out a simulator to see which issues pop up for the variety of types of colorblindness. Since that's a spectrum, might be nice to provide some guidance for things like difficulty for roads projects v buildings projects. Maybe we can point people towards one for example.

**Comment by AlittaAbraham** [comment]
There was a suggestion from a user to make the colors brighter and make it less transparent to help people with color blindness.

**Comment by Hagellach37** [comment]
I've looked into this and made a short video. But will also try to use some simpler icons:


https://user-images.githubusercontent.com/7045979/122436110-d88c8b00-cf98-11eb-9797-cd96cf25c058.mp4

**Comment by Hagellach37** [comment]
and here another option:


https://user-images.githubusercontent.com/7045979/122440141-949b8500-cf9c-11eb-96c5-2752a7258f74.mp4

@luisgregson @adrian7031  I know it's quite some time ago that you commented on this issue and I'm not if you are still around. But it would be great to hear your thoughts.

If you provide me some of your icons I can add them and we can give it a test in the app. :)

**Comment by kopitek8** [comment]
Attaching here the proposed solution for the issue: https://docs.google.com/document/d/10EBmxJDjAfkL4Cqm-DPgKjnBrmGNcr483jKMLgYm2tc/edit#heading=h.cj5w6puok8mu

**Comment by kopitek8** [comment]
## Updates

- Completed accessibility issue in the following
  - Mapping screen
  - Feature under settings (off by default)
